From: Reinier Haasjes <reinier@haasjes.com>
Date: Fri, 17 Aug 2012 11:29:39 +0200
Subject: no shipped init script

We ship our own init.d script and do not want to see the upstream default
for Debian installed into DESTDIR.
---
 Makefile |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 0e96136..3de13db 100644
--- a/Makefile
+++ b/Makefile
@@ -84,9 +84,9 @@ ifeq ($(shell echo "A${RPM_BUILD_ROOT}"),A)
 	@cp doc/README ${DESTDIR}${dirdoc}
 	@cp doc/LICENSE ${DESTDIR}${dirdoc}
 	@cp doc/HOWTO  ${DESTDIR}${dirdoc}
-	@echo "Installing Debian-style init.d"
-	@mkdir -p ${DESTDIR}${diretc}init.d
-	@cp doc/${PROJECT}.init.debian ${DESTDIR}${diretc}init.d/${PROJECT}
+#	@echo "Installing Debian-style init.d"
+#	@mkdir -p ${DESTDIR}${diretc}init.d
+#	@cp doc/${PROJECT}.init.debian ${DESTDIR}${diretc}init.d/${PROJECT}
 else
 	@echo "Installing Redhat-style init.d"
 	@mkdir -p ${DESTDIR}${diretc}init.d
